### PR TITLE
Start of the databasing

### PR DESCRIPTION
**Pull request by HannahSMead**
This is the basic table structure we need within our lovely database. More methods and functions to come!

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+engine = create_engine('postgresql://@localhost:5432/myDB', echo = True)
+
+class Account(Base):
+    """
+    The main user class for ReeDistribute Login.abs
+    Attributes: account_id (int), name (str), email (str), consumer_key (str), anon (int), admin (int)
+    """
+    __tablename__='accounts' #Store all account objects in accounts
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account account_identification
```
Comment (flllra) [inline db.py:21]: is this what we will be calling when people log in?

File: db.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        return s
+
+class Donor(Base):
+    """
+    The class for Donors
+    Attributes:  account_id (int), closed_requests (Int), money_given (int)
+    """
+    __tablename__= 'donors' #Store all donor objects in the table donors
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account identification
```
Comment (flllra) [inline db.py:69]: is this only in the case where the donors themselves would have accounts too?

File: db.py (hunk 1 above)
Comment (HannahSMead) [reply db.py:21]: Nope, I haven't yet written that part yet, but soon to come as methods in the donor and student classes or as a normal python function. I will be testing to figure out which one is better/functions at all.

File: db.py (hunk 2 above)
Comment (HannahSMead) [reply db.py:69]: No, I was creating accounts for donors so we can record the statistics R4R suggested.

Comment (HannahSMead) [reply db.py:69]: But also yes, since they would need accounts to be able to have that information tracked for thank you notes and stuff!

**Pull request closed without merging**